util/topsql/stmtstats/stmtstatstest/stmtstats_test.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stmtstatstest
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/pingcap/tidb/config"
+	"github.com/pingcap/tidb/parser"
+	"github.com/pingcap/tidb/session"
+	"github.com/pingcap/tidb/sessionctx/variable"
+	"github.com/pingcap/tidb/store/mockstore"
+	"github.com/pingcap/tidb/testkit"
+	"github.com/pingcap/tidb/util/topsql/stmtstats"
+	"github.com/stretchr/testify/assert"
+	"github.com/tikv/client-go/v2/testutils"
+)
+
+func TestExecCount(t *testing.T) {
```

Comment:
I have added the integration test of ExecCount, PTAL~ @breeswish @crazycs520 @zhongzc

Reply:
How about just remove `stmtstatstest` pkg and rename the test `pkg` name to `stmtstats_test`

Reply:
This will cause `stmtstats` to depend on higher-level packages such as `session`, this will lead to circular dependencies.